### ISSUE TITLE
Use local .envfile instead of /tmp/envfile

### ISSUE THREAD

**Issue by ltvan** [ByneappLLC/flutter_config#2]
You can use local .envfile instead of /tmp/envfile. I copy here my patch for react-native-config. I haven't try to update your code yet.

```diff

 defaultEnvFile = ".env"
+envFilePath = File.join(Dir.pwd, "../../../.envfile")
 
 # pick a custom env file if set
-if File.exists?("/tmp/envfile")
+if File.exists?(envFilePath)
+  custom_env = true
+  file = File.read(envFilePath).strip
+elsif File.exists?("/tmp/envfile")
   custom_env = true
   file = File.read("/tmp/envfile").strip
 else
@@ -73,7 +77,11 @@ path = File.join(ENV["BUILD_DIR"], "GeneratedInfoPlistDotEnv.h")
 File.open(path, "w") { |f| f.puts info_plist_defines_objc }
 
 if custom_env
-  File.delete("/tmp/envfile")
+  if File.exists?(envFilePath)
+    File.delete(envFilePath)
+  elsif File.exists?("/tmp/envfile")
+    File.delete("/tmp/envfile")
+  end
 end
```

**Comment by ltvan** [comment]
An alternative way is to use SCHEME_NAME as a default suffix.

**Comment by edTheGuy00** [comment]
Hi, Thanks.

This seems like a better approach than using the local `tmp` directory. I'll try to add this in

**Comment by edTheGuy00** [comment]
I sort of halfway got this method working on [this branch](https://github.com/ByneappLLC/flutter_config/tree/update-ios-method) , but I was unable to generate the `tmp.xcconfig` required for the variables to available in the `Info.plist`

I think the main issue is that I am trying to place the `.envfile` in the iOS directory instead of the root directory, I think this is a better approach since that file is only used in iOS, 

Anyways, I gave up for now. Might come back to this later. But placing the `envfile` in `/tmp/envfile` works perfectly fine for now

**Comment by edTheGuy00** [comment]
fixed in #9
